deps/src/brainflow-boardcontroller/brainflow_boards.cpp

```diff
@@ -271,7 +271,18 @@ json brainflow_boards_json = {
             {"eeg_channels", {1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32}},
             {"emg_channels", {1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32}},
             {"ecg_channels", {1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32}}
-        }}
+        }},
```

Comment:
Ask @Andrey1994 for permission to add this to his repo https://github.com/brainflow-dev/brainflow otherwise this will be overwritten.

Reply:
@dioptre  
I have updated my code with the latest version please checkout to the latest commit of this PR. And I have read about brainflow, Indeed it's having prominent features which can be beneficial for us, we would certainly implement it in our future work after having ample understanding of it, but as of now, I would request you to merge this code in your main code after a peer review.